### DISCUSSION TITLE
ControlNet v1.1.409 to work with backhand diffusers

### DISCUSSION THREAD

**Discussion by ShmuelRonen** [vladmandic/sdnext#2170] [Q&A]
Although I installed ControlNet v1.1.409 the checkup box is not check.
I can understand from it that this option is not yet enabled/ implemented in backhand diffusers mode.
Also the new version of Deforum Not working as well.

Are there a schedule for that?

**Comment by rodrigoscoelho** [comment]
same problem here

**Comment by vladmandic** [comment]
re: controlnet - its been covered in 20+ messages and threads. yes, its coming, but please please please search first.
